### PR TITLE
feat: add notification

### PR DESCRIPTION
**Pull request by conwnet**
According #51, It seems there are some users get confused with GitHub1s, maybe we should add a notification to explain it? Please help me to review the notification message. Is it clear enough? Is there any grammatical error?😂

### CODE REVIEW

File: src/vs/github1s/notification.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import 'vs/css!./notification';
+
+const NOTIFICATION_STORAGE_KEY = 'GITHUB1S_NOTIFICATION';
+// change it if a new notification should be shown
+const NOTIFICATION_STORAGE_VALUE = '20210212';
+
+const notification = {
+	title: 'ATTENTION: This page is NOT officially provided by GitHub.',
+	content: 'GitHub1s is open source project, not officially provided by GitHub.',
+	link: 'https://github.com/conwnet/github1s',
```
Comment (conwnet) [inline src/vs/github1s/notification.ts:15]: there is the notification message

**Comment by conwnet** [comment]
This is the screenshot.

image

File: src/vs/github1s/notification.ts (hunk 1 above)
Comment (dkd-thoma) [reply src/vs/github1s/notification.ts:15]: Suggestion for content: `GitHub1s is an open source project, which is not officially provided by GitHub.`

File: src/vs/github1s/notification.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+import 'vs/css!./notification';
+
+const NOTIFICATION_STORAGE_KEY = 'GITHUB1S_NOTIFICATION';
+// change it if a new notification should be shown
+const NOTIFICATION_STORAGE_VALUE = '20210212';
+
+const notification = {
+	title: 'ATTENTION: This page is NOT officially provided by GitHub.',
+	content: 'GitHub1s is open source project, not officially provided by GitHub.',
```
Comment (dkd-thoma) [inline src/vs/github1s/notification.ts:14]:
```suggestion
	content: 'GitHub1s is an open source project, which is not officially provided by GitHub.',
```

**Review by dkd-thoma — CHANGES_REQUESTED** [review]
Suggestion for content

File: src/vs/github1s/notification.ts (hunk 1 above)
Comment (conwnet) [reply src/vs/github1s/notification.ts:15]: Thanks~

**Review by Siddhant-K-code — APPROVED** [review]

**Pull request merged**